static/app/views/nav/prefersStackedNav.tsx

```diff
@@ -1,12 +1,23 @@
 import ConfigStore from 'sentry/stores/configStore';
+import type {Organization} from 'sentry/types/organization';
+import useOrganization from 'sentry/utils/useOrganization';
 import {useUser} from 'sentry/utils/useUser';
 
-export function prefersStackedNav() {
-  return ConfigStore.get('user')?.options?.prefersStackedNavigation ?? false;
+export function prefersStackedNav(organization: Organization) {
+  return (
+    organization.features.includes('enforce-stacked-sidenav') ??
```

Comment:
This isn't quite right, `??` only defers to the latter statement if it's null/undefined so this will never fall back to user config